modules/ROOT/pages/_partials/include-general-config.adoc

```diff
@@ -96,7 +102,35 @@ Valid values and arguments:
 | Alias By | You can create a label for your metric using a literal string (such as `my string`) or supported variables (for example, `$tag_worker_id` or `$tag_env_id`).
 |===
 
-After you complete your configuration, you can click Save Changes. To exit the configuration and return to the dashboard, simply click the `X`.
+[%header,cols="1,4"]
+|===
+2+| Advanced Mode Settings for APIs
+| From a|
+
+`http_summary_metric` 
+
+For a list of advanced metrics, see xref:dashboard-config-ref.adoc#metrics_advanced[Advanced Mode Metrics]
+| Where a|
+Provides conditional logic for filtering the *From* data source you selected.
+
+You can select one or more filters tags (such as `app_id`, `org_id`, `api_version_id`, `method`, `status code`). Example: `Where status_code = 404`.
+
+|Select a|
+
+For creating a function that selects a field (such as `request_size_average`,`response_time_max`, or `response_time_sum`) from the HTTP summary and applies an operation to it (such as the `mean` of the `response_time_average`). You can add one or more `field` settings.
+|Group by a|
+
+For grouping the data by a selected time interval (such as `$__interval`, `1s`, `10m`, `1h`), filter tag (such as `tag(app_id)`), and fill options (such as `none`, `null`, `0`, `previous`, `linear`).
+|Alias by a|
+
+You can create a label for your metric using a literal string (such as `my string`) or supported variables (for example, `$tag_app_id`). The alias must always begin with `$tag`. The syntax is `$tag_yourTagName`.
```

Comment:
Please use in the example api_id instead app_id. You can only use app_id when choose Application panel.